src/Transporters/HttpTransporter.php

```diff
@@ -113,7 +113,7 @@ private function throwIfJsonError(ResponseInterface $response, string|ResponseIn
             return;
         }
 
-        if ($response->getheader('Content-Type')[0] !== 'application/json; charset=utf-8') {
+        if (! in_array($response->getheader('Content-Type')[0], ['application/json', 'application/json; charset=utf-8'])) {
```

Comment:
A simpler fix would be to do this. But I'm uncertain if there consequences.

```suggestion
        if ($response->getheader('Content-Type')[0] !== 'application/json') {
```